lasy/profiles/longitudinal/longitudinal_profile_from_data.py

```diff
@@ -25,12 +25,15 @@ class LongitudinalProfileFromData(LongitudinalProfile):
 
         datatype : string
             The domain in which the data has been passed. Options
-            are 'spectral' and 'temporal'
+            are 'spectral' and 'temporal'. Spectral data may either be,
+            a) Wavelength dependent (lambda): data["axis_is_wavelength"] = True or
+            b) Angular freuqency dependent (omega): data["axis_is_wavelength"] = False
 
```

Comment:
This option probably warrants a separate entry, here's a suggestion for this.
```suggestion

        axis_is_wavelength : boolean (optional, default True)
            If True, the axis represents wavelength in [m] (SI).
            If False, it represents angular frequency in [rad/s] (SI).

```